src/components/Carousel/index.js

```diff
@@ -76,6 +76,7 @@ export default class Carousel extends PureComponent {
     sliderRef: PropTypes.func,
     transition: PropTypes.string,
     variableWidth: PropTypes.bool,
+    color: PropTypes.string,
```

Comment:
I think this is supposed to be `colorForThatFantasticFunkInPeekMode`

Reply:
dammit!  good catch